tests/plugin/common.py

```diff
@@ -81,9 +60,9 @@ def run_tests(
 def run_with_allure(*args, **kwargs):
     """Run tests with fake allure logger"""
 
-    allure_report = AllureMemoryLogger()
-    with fake_logger("allure_pytest.plugin.AllureFileLogger", allure_report):
-        addopts = kwargs.get("additional_opts", [])
-        addopts.extend(["--alluredir", "::in-memory::"])
-        kwargs["additional_opts"] = addopts
-        return run_tests(*args, **kwargs), allure_report
+    addopts = kwargs.get("additional_opts", [])
+    addopts.extend(["--alluredir", "allure-results"])
```

Comment:
In this case `allure-result` directory is relative to `pytester` working directory?

Reply:
yep